client/components/app.jsx

```diff
@@ -2,11 +2,13 @@ import React from 'react';
 import axios from 'axios';
 import styled from 'styled-components';
 import MarketOrder from './marketOrder.jsx';
+import DropDown from './dropdown.jsx';
 import Wrapper from './styles/mainWrapper/wrapper';
 import InputWrapper from './styles/inputWrapper/inputWrapper';
```

Comment:
I really like how you separated the styling from the component-building files. Definitely something I will do when I clean up/refactor my code!